_posts/2016-09-23-apple-echo.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author: tjdraper
+title: "Apple Echo"
+categories: link
+custom_type: link
+link_url: http://www.bloomberg.com/news/articles/2016-09-23/apple-said-to-step-up-plans-for-echo-style-smart-home-device-itfnod11
+tags:
+  - Amazon Echo
+  - Smart Home
+date: 2016-09-23 13:55
+---
+Mark Gurman and Ian King on Bloomberg:
+
+> Apple Inc. is pressing ahead with the development of an Echo-like smart-home device based on the Siri voice assistant, according to people familiar with the matter.
+>
+>Started more than two years ago, the project has exited the research and development lab and is now in prototype testing, said the people, who asked not to be identified discussing unannounced Apple projects. Like Amazon Inc.’s Echo, the device is designed to control appliances, locks, lights and curtains via voice activation, the people said. Apple hasn’t finalized plans for the device and could still scrap the project.
+
+As a fan of Apple products and Apple the company, it is frustrating to continue to see them lose focus. I know it’s easy to say, "this would never have happened if Steve were alive" and if and when I say that, I don't say it lightly. And I'm not ready to say that now, actually. But I will say it is improbable that this would be a direction the company would have gone under Steve. The reason I say that is because one of Steve's ideals — one of the things he championed — was razor sharp focus. [Focus is about saying no].
```

Comment:
Hmm. I can’t remember which styleguide it comes from, but only one of them does it. I remember because I looked it up looking for a valid reason I could space around them because I think it looks infinitely better. But I will abide by The Bold Report style.